src/server/static/definitions/fra/2025/es/tad.md

```diff
@@ -130,7 +130,7 @@ En este contexto, predominantemente significa que se espera que los árboles pla
 
 > **Bosques plantados** no clasificados como **plantación forestal**.
 
-## 1e Categorías específicas de los bosques
+## 1c Categorías específicas de los bosques <!-- section:1c -->
```

Comment:
The '1e' label was wrong. "Categorías específicas de los bosques" corresponds to 1c.

![image](https://github.com/openforis/fra-platform/assets/41337901/98070981-3e5e-4d1a-94aa-2d14cdc76da1)